.github/workflows/build.yaml

```diff
@@ -0,0 +1,65 @@
+# name of the work flow
+name: Build
+on: [pull_request]
```

Comment:
Do we want to create a release for every action on every PR? I think this should be:
```
on:
  push:
    branches:
      - main
```

Reply:
Yes, I am also thinking on the same line.